upload-react/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```

Comment:
Maybe we could change the README a bit from this boilerplate? For example describe what are you doing in the example, what Bee(JS) APIs are you using, and how? Example of my README for the MetamaskSigning: https://github.com/ethersphere/bee-js/pull/168/files#diff-9576d5f38c5e309082bc2573cef0e58eb5bfd6e1a44239aba036ee64686b12e2

Reply:
Sorry I haven't seen this. This sounds like a good idea and I'll add it but will not be able to do so until 3PM.

Reply:
Sounds good